scripts/decredPoliteia.md

```diff
@@ -1,25 +1,29 @@
-#Decred in Depth: Politeia
+# Decred in Depth: Politeia
 
 ### Keywords
 `proposal`, `Timestamped`, `versioned`, `self-governance`, `immutable`, `off-chain storage`, `git`, `chain-anchored`
 
-Named after the greek term meaning ‘a system of government’, Politeia is the Decred proposal system a social media application where Decred stakeholders submit proposals for projects to upgrade and market the Decred network. 
+Named after the greek term meaning ‘a system of government’, Politeia is the Decred proposal system: a chain-anchored social media application where Decred stakeholders can take part in deciding which Decred improvement projects recieve funding from the development fund. 
 
-Politeia is special in that it is designed to be an immutable and cryptographically verifiable public record of Decred’s governance-data. This means that once a proposal is posted to Politeia, the record of that proposal’s posting can never be altered or deleted. In addition, records of proposals cannot be forged. The time ordered and independently verifiable nature of Politeia ensures that attempts to manipulate Decred’s governance are significantly more difficult than if a traditional website or social media application were in use.
+Stakeholder submitted project proposals, proposal comments, and votes make up Decred's governance data hosted on Politeia. Newly submitted content is episodically anchored to the Decred blockchain creating an immutable and cryptographically verifiable public record of Decred's governance-data. Once a proposal is posted to Politeia, the record of it's posting can never be altered or deleted, nor can proposals be forged. 
 
-From a technical standpoint, Politeia is an off-chain storage system comprised  of Git, a popular revision control system, and cryptographic timestamping via DecredTime, a custom server application for storing timestamps on the Decred blockchain.
+Anchoring Politeia to the blockchain ensures that attempts to manipulate Decred's governance are significantly more difficult than if a traditional website or social media application were in use.
 
-Git tracks and stores changes to Politeia as revisions. These stored revisions make up the public record of Decred governance on Politeia. DecredTime takes unique identifiers of these revisions, called hashes, and anchors them to the Decred blockchain, creating a time sequence that is computationally impossible to recreate.
+To achieve this chain-anchoring, Politeia's off-chain storage system makes use of Git (a popular revision control system), and DecredTime (a custom server application for storing timestamps on the Decred blockchain.)
```

Comment:
It's [dcrtime](https://github.com/decred/dcrtime), haven't seen "DecredTime" earlier. It may work and easier to pronounce, I'm just noting for completeness.

Reply:
DecredTime is spelled out for voice over artist's benefit

Reply:
unless its supposed to be pronounced Dee Cee Ar?

Reply:
Dee Cee Ar Time is more correct at the moment since nobody used DecredTime earlier. But for the benefit of the narrator I agree DecredTime is better (-1 syllable) and is not too incorrect.